### PR TITLE
Fixed bug when homepage couldn't be loaded if DB is epmty.

### PR DESCRIPTION
**Pull request by SerhiyToroniy**

### CODE REVIEW

File: sports_hub/sports_hub/Controllers/HomeController.cs [hunk 1]
```diff
@@ -54,15 +54,13 @@ public async Task<IActionResult> IndexAsync()
...
                 ViewBag.MainArticleCategoryName = mainArticle.CategoryName;
+            }
 
-                ViewBag.AvailableLanguages = _context.Languages.ToList();
+            ViewBag.AvailableLanguages = _context.Languages.ToList();
 
-                //Setting breakdown articles
-                List<List<Article>> breakdownAtrciles = await _articles_provider.GetArticlesForBreakdownAsync();
-                ViewBag.breakdownArticles = breakdownAtrciles;
-                HttpContext.Session.SetInt32("LeftBarIndex", 0);
```
Comment (XashDM) [inline sports_hub/sports_hub/Controllers/HomeController.cs:63]: Why this is removed? @Yurii-Vipshovsky do we still need it?

**Review by XashDM — APPROVED** [review]
Need feedback from Yurii - then we can merge

**Review by XashDM — APPROVED** [review]

**Review by XashDM — CHANGES_REQUESTED** [review]
Your code isn't compiling

File: sports_hub/sports_hub/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -61,41 +60,42 @@
...
+<div class="breakdown-small" onclick="articleClick(@breakdownSection[i].Id)">
+    <img class="news-picture" src="data:image/png; base64, @Convert.ToBase64String(breakdownSection[i].Image)" />
+    <div>
+        <p class="mini-news-name">@breakdownSection[i].Headline</p>
+        <p class="mini-news-info">@breakdownSection[i].Caption</p>
+    </div>
+    <img class="svg" src="~/images/Arrow.svg" />
+</div>}}
+</div>} } }
+            <script>
```
Comment (ZarechOleksii) [inline sports_hub/sports_hub/Views/Home/Index.cshtml:93]: Move to JS file and call as a function

**Review by ZarechOleksii — APPROVED** [review]

File: sports_hub/sports_hub/Controllers/HomeController.cs (hunk 1 above)
Comment (Yurii-Vipshovsky) [reply sports_hub/sports_hub/Controllers/HomeController.cs:63]: It's not my task so I don't know

Comment (Yurii-Vipshovsky) [reply sports_hub/sports_hub/Controllers/HomeController.cs:63]: But this code we need
HttpContext.Session.SetInt32("LeftBarIndex", 0);

Comment (Yurii-Vipshovsky) [reply sports_hub/sports_hub/Controllers/HomeController.cs:63]: we can merge this code

**Review by XashDM — APPROVED** [review]
Looks good

**Pull request merged**